pkgs/by-name/pi/pixelorama/package.nix

```diff
@@ -1,64 +1,73 @@
-{ lib
-, stdenv
-, alsa-lib
-, autoPatchelfHook
-, fetchFromGitHub
-, godot3-headless
-, godot3-export-templates
-, libGLU
-, libpulseaudio
-, libX11
-, libXcursor
-, libXi
-, libXinerama
-, libXrandr
-, libXrender
-, nix-update-script
-, udev
+{
+  lib,
+  stdenv,
+  alsa-lib,
+  autoPatchelfHook,
+  fetchFromGitHub,
+  godot_4,
+  godot_4-export-templates,
+  libGL,
+  libpulseaudio,
+  libX11,
+  libXcursor,
+  libXext,
+  libXi,
+  libXrandr,
+  vulkan-loader,
+
+  nix-update-script,
+  udev,
 }:
 
 let
   preset =
     if stdenv.isLinux then
-      if stdenv.is64bit then "Linux/X11 64-bit"
-      else "Linux/X11 32-bit"
-    else if stdenv.isDarwin then "Mac OSX"
-    else throw "unsupported platform";
-in stdenv.mkDerivation (finalAttrs: {
+      if stdenv.is64bit then "Linux/X11 64-bit" else "Linux/X11 32-bit"
+    else if stdenv.isDarwin then
+      "Mac OSX"
+    else
+      throw "unsupported platform";
+
+  godot_version_folder = lib.replaceStrings [ "-" ] [ "." ] godot_4.version;
+in
+stdenv.mkDerivation (finalAttrs: {
   pname = "pixelorama";
-  version = "0.11.4";
+  version = "1.0.1";
 
   src = fetchFromGitHub {
     owner = "Orama-Interactive";
     repo = "Pixelorama";
     rev = "v${finalAttrs.version}";
-    sha256 = "sha256-VEQjZ9kDqXz1hoT4PrsBtzoi1TYWyN+YcPMyf9qJMRE=";
+    hash = "sha256-lfim5ZiykOhI1kgsu0ni2frUVHPRIPJdrGx6TuUQcSY=";
+    fetchSubmodules = true;
```

Comment:
```suggestion
```
Would be good to avoid this. If some dependency is missing, package that.

Reply:
Ah yes, I added this while debugging an issue, but it's actually not needed.